sdk/ai/ai-inference-rest/test/public/chatCompletions.spec.ts

```diff
@@ -226,4 +226,62 @@ describe("chat test suite", () => {
       timeout: 50000
     });
 
+  it("multi-turn chat test", async function () {
+    const messages = [
+      { role: "system", content: "You are a helpful assistant answering questions regarding length units." },
+      { role: "user", content: "How many feet are in a mile?" },
+    ];
+    const response = await client.path("/chat/completions").post({
+      body: { messages }
+    });
+
+    assert.isFalse(isUnexpected(response));
+
+    const completion = response.body as ChatCompletionsOutput;
+    assert.isDefined(completion);
+    assert.isNotEmpty(completion.choices);
+    assert.isDefined(completion.choices[0].message);
+    assert.isDefined(completion.choices[0].message.content);
+
+    const assistantMessage = completion.choices[0].message.content as string;
+    assert.isTrue(assistantMessage.includes("280"));
+    messages.push({ role: "assistant", content: assistantMessage });
+    messages.push({ role: "user", content: "and how many yards?" });
+
+    const secondResponse = await client.path("/chat/completions").post({
+      body: { messages }
+    });
+
+    assert.isFalse(isUnexpected(secondResponse));
+
+    const secondCompletion = secondResponse.body as ChatCompletionsOutput;
+    assert.isDefined(secondCompletion);
+    assert.isNotEmpty(secondCompletion.choices);
+
+    assert.isDefined(secondCompletion.choices[0].message);
+    assert.isDefined(secondCompletion.choices[0].message.content);
+
+    const yardsMessage = secondCompletion.choices[0].message.content as string;
+    assert.isTrue(yardsMessage.includes("760"));
+  },
+    {
+      timeout: 80000
+    });
+
+  it("chat auth error test", async function () {
+    client = await createModelClient("dummy", recorder);
+    const response = await client.path("/chat/completions").post({
+      body: {
+        messages: [
+          { role: "user", content: "How many feet are in a mile?" },
+        ]
+      }
+    });
+
+    assert.isTrue(isUnexpected(response));
+  },
+    {
+      timeout: 50000
```

Comment:
timeout could also be configured once in the vitest config module.

Reply:
[Updated](https://github.com/Azure/azure-sdk-for-js/pull/30520/commits/036be1444ae044aef9fd7de4ea65becd6b23e5ca)